k8s/src/main/scala/io/buoyant/k8s/Api.scala

```diff
@@ -0,0 +1,187 @@
+package io.buoyant.k8s
```

Comment:
Hm... this is probably more trouble than it's worth (and if there really is a v2 api we can just create a k8s-v2 package...)